src/nbl/video/IGPUCommandBuffer.cpp

```diff
@@ -1774,6 +1895,144 @@ bool IGPUCommandBuffer::resolveImage(const IGPUImage* const srcImage, const IGPU
     return resolveImage_impl(srcImage, srcImageLayout, dstImage, dstImageLayout, regionCount, pRegions);
 }
 
+bool IGPUCommandBuffer::setRayTracingPipelineStackSize(uint32_t pipelineStackSize)
+{
+    if (!checkStateBeforeRecording(queue_flags_t::COMPUTE_BIT,RENDERPASS_SCOPE::OUTSIDE))
+        return false;
+    if (m_boundRayTracingPipeline != nullptr && m_boundRayTracingPipeline->getCachedCreationParams().dynamicStackSize)
+    {
```

Comment:
try `!m_boundRayTracingPipeline || m_boundRayTracingPipeline->getCachedCreationParams().dynamicStackSize` operator short circuits so its safe to do

Because right now you have a corner case not convered, which is a newly begun commandbuffer with no pipeline bound yet, you'll get no error.

Reply:
done by me